### ISSUE TITLE
Proplem occurred when training yolo v4 for custom dataset from scratch

### ISSUE THREAD

**Issue by SOWENKIN** [AlexeyAB/darknet#5336]
![error](https://user-images.githubusercontent.com/11489552/80301359-4a72a580-87d6-11ea-9d32-7c6f2c463972.png)
"Class: 0.000000, Obj: 0.000000, No Obj: 0.000000, .5R: 0.000000, .75R: 0.000000" all the time.
```
[net]
batch=64
subdivisions=32
width=608
height=608
channels=3
momentum=0.949
decay=0.0005
angle=0

learning_rate=0.00261
burn_in=1000
max_batches = 500500
policy=steps

steps=98418,196836,295254,393672,450000
scales=.1,.1,.5,.5,.5

mosaic=1
blur=1
```

```
[convolutional]
size=1
stride=1
pad=1
filters=90
activation=linear


[yolo]
mask = 0,1,2,3,4
anchors = 12,12,  23,22,  27,35,  41,30,  32,73,  54,54,  89,34,  77,72,  61,105, 125,68,  97,100, 145,114, 107,158, 177,167, 219,232
classes=13
num=15
jitter=.3
ignore_thresh = .7
truth_thresh = 1
scale_x_y = 1.2
iou_thresh=0.213
cls_normalizer=1.0
iou_normalizer=0.07
iou_loss=ciou
nms_kind=greedynms
beta_nms=0.6
```

**Comment by WongKinYiu** [comment]
For training custom dataset from scratch, it is better to decrease learning rate and increase burn_in.

**Comment by SOWENKIN** [comment]
> For training custom dataset from scratch, it is better to decrease learning rate and increase burn_in.

Thanks, problem solved!
